pallets/messages/src/tests/other_tests.rs

```diff
@@ -68,24 +60,9 @@ fn populate_messages(
 /// Helper function to generate message payloads
 ///
 /// # Arguments
-/// * `num_items` - Number of message to include in payload
```

Comment:
This seemed overly complicated--a payload is just an array of bytes; why all the JSON stuff?